clients/browser/index.html

```diff
@@ -431,25 +442,61 @@
     // Safari quirks: don't use the same var var name in global scope as id of html element
     var overlay_ = document.querySelector('#overlay');
 
-    Nimiq.init(function($) {
-        window.$ = $;
-        $.network.connect();
-
-        overlay_.style.display = 'none';
-        window.ui = new MinerUI($);
-    }, function(code) {
-        switch (code) {
-            case Nimiq.ERR_WAIT:
-                overlay_.style.display = 'block';
-                break;
-            case Nimiq.ERR_UNSUPPORTED:
-                alert('Browser not supported');
-                break;
-            default:
-                alert('Nimiq initialization error');
-                break;
-        }
-    });
+    function dumpWallet() {
+      alert($.wallet.dump());
+    }
+
+    function loadWallet() {
+      var walletSeed = document.getElementById('walletSeed').value;
+      var options = {
+        walletSeed: walletSeed
+      };
+
+      Nimiq.init(function($) {
+          window.$ = $;
+          $.network.connect();
+
+          overlay_.style.display = 'none';
+          window.ui = new MinerUI($);
+      }, function(code) {
+          switch (code) {
+              case Nimiq.ERR_WAIT:
+                  overlay_.style.display = 'block';
+                  break;
+              case Nimiq.ERR_UNSUPPORTED:
+                  alert('Browser not supported');
+                  break;
+              default:
+                  alert('Nimiq initialization error');
+                  break;
+          }
+      },
+      options
+    );
+
+    }
+
+    function createWallet() {
+      Nimiq.init(function($) {
```

Comment:
`createWallet()` doesn't create a Wallet if there is already one in index db. Choose a different name.